### ISSUE TITLE
rainbow-hsl doesnt appear to work.

### ISSUE THREAD

**Issue by henryw374** [jolby/colors#13]
Hi, just installed and tried rainbow-hsl. fails on an assert in inclusive-seq that the arg should be less than 1. 

I was struggling to follow the inclusive-seq fn, so, in the meantime wrote my own:

``````
(defn rainbow-hsl [numcolors & opts]
  (let [opts (merge {:s 50.0 :l 70.0 :start 0 :step (/ 360 numcolors)}
                    (when opts (apply assoc {} opts)))]
    (->>
      (range 1 (inc numcolors))
      (map #(mod (+ (:start opts) (* % (:step opts))) 360))
      (map #(rgb/create-color :h (float %) :s (:s opts) :l (:l opts)))
      (map rgb/rgb-hexstr))))```
``````

**Comment by shen-tian** [comment]
Second that. 

    (rainbow-hsl 10)

doesn't work. Looking at the `inclusive-seq` source, think the inequality in this `assert` is the wrong way around?

    (assert (< n 1)

@jolby might have moved on from the library though. Time to fork and cleanup?

**Comment by henryw374** [comment]
I've recently forked to get this lib compiling under clojure 1.9 here. https://github.com/henryw374/colors/

happy to add any pull requests to that if you want. I'm unlikely to do much actual dev on this at the moment though so might be better if you fork and manage releases?

**Comment by shen-tian** [comment]
Having had a look around, this is a more active library: https://github.com/thi-ng/color

**Comment by jolby** [comment]
The broken inclusive-seq function should be fixed as of 1.0.6. I think that will allow rainbow-hsl functions to work but I haven't really tested them.
